### PR TITLE
[rfw] Run tests for package:rfw on stable as well.

### PR DESCRIPTION
**Pull request by Hixie**
For non-master-channel builds, the run_tests.sh script runs but
doesn't bother checking the actual coverage, since I expect Dart's
coverage logic to diverge in master and so we'd have to have
per-channel covered line counts, and that way lies madness.

No version change: Only affects how tests are run, not a public-facing change.
No CHANGELOG change: Only affects how tests are run, not a public-facing change.

### CODE REVIEW

**Comment by stuartmorgan** [comment]
See also https://github.com/flutter/packages/pull/886, where I'm turning on the missing unit tests on stable, but had to exclude rfw because of goldens. I don't know if you want to do something more clever within the rfw tests themselves to only exclude the goldens?

**Comment by Hixie** [comment]
```patch
-    print('Tests passed. (Coverage verification skipped; not on master channel.)');
+    print(
+        'Tests passed. (Coverage verification skipped; not on master channel.)');
```

sigh

**Comment by Hixie** [comment]
> See also #886, where I'm turning on the missing unit tests on stable, but had to exclude rfw because of goldens. I don't know if you want to do something more clever within the rfw tests themselves to only exclude the goldens?

any idea why the tests didn't fail? i didn't do anything to avoid stable for the goldens...

**Comment by stuartmorgan** [comment]
> i didn't do anything to avoid stable for the goldens...

I did, it's just ugly:
https://github.com/flutter/packages/pull/886/files#diff-62587956f943bb2503db7bc6dd27d0d888074a1c0ecaab3f570ad611aff0f7bbR84-R90

**Comment by Hixie** [comment]
right but that hasn't landed yet, right? so why did my precommits not fail?

**Comment by stuartmorgan** [comment]
Oh, *your* precommits are fine because of https://github.com/flutter/flutter/issues/97233 (which is why my PR is fixing).

**Comment by Hixie** [comment]
oh, haha. Ok let me figure out some way to skip the tests properly. in the meantime feel free to land 866.

**Comment by Hixie** [comment]
@stuartmorgan PTAL

File: packages/rfw/test_coverage/bin/test_coverage.dart [hunk 1]
```diff
@@ -15,39 +15,32 @@ const String lastUpdate = '2021-08-30';
...
   );
   if (result.exitCode != 0) {
     print(result.stdout);
     print(result.stderr);
     print('Tests failed.');
+    // leave coverage directory around to aid debugging
     exit(1);
   }
 
+  if (Platform.environment['CHANNEL'] != 'master') {
```
Comment (stuartmorgan) [inline packages/rfw/test_coverage/bin/test_coverage.dart:36]: Did you want the "no channel set" condition here as well for local runs?

**Review by stuartmorgan — APPROVED** [review]
LGTM

**Comment by Hixie** [comment]
Sorry, other efforts took my attention away from my PRs. I'll return to them in due course. Feel free to land this for me in the meantime if you want (looks like it'll need merging first).

**Pull request merged**